add-on/panels/css/panel.css

```diff
@@ -207,15 +195,39 @@ body {
 }
 
 .room-list {
-  /* xxx  not sure why flex needs the 3 value setting
-    but setting flex to just 1, the whole tab including the new room is scrollable.
-    seems to not like the 0% of the default setting - may be FF bug */
-  flex: 1 1 0;
   overflow-y: auto;
   overflow-x: hidden;
-  display: flex;
   flex-flow: column nowrap;
   width: 100%;
+  /* max-height is determined by the max number of rooms shown,
+   * which are 5 @ 28px each + padding */
+  max-height: 144px;
```

Comment:
Please use calc() here just to make it more clear, and the comment can probably be less verbose.